test/cli.js

```diff
@@ -60,16 +60,6 @@ module.exports = [
       return code === 1 && stderr.toString().indexOf('fail.ts:2:1') !== -1;
     }
   },
-  {
-    args: ["run", "test/fixtures/ts-error2/ts-error.ts"],
-    expect (code, stdout, stderr) {
-      return code === 1 && stderr.toString().indexOf('ts-error.ts(3,16)') !== -1 && stderr.toString().split('\n').length < 11;
-    }
-  },
-  {
-    args: ["run", "-t", "test/fixtures/with-type-errors/ts-error.ts"],
-    expect: { code: 0 }
```

Comment:
Why were these 3 tests removed?

Reply:
These tests are specific to verify tsc to fail if given typescript file contains error: for example, test verifies output 

```
code === 1 && stderr.toString().indexOf('ts-error.ts(3,16)') !== -1
```

to check tsc's error message contains error line to 3:16 where it have type-related error:

```
//3:16 points 'Y', since there's no corresponding types
function p (x: Y) { 
```

which doesn't have any meaningful behavior in swc as it strips out typescript and does not emit any type check failures.

Reply:
Ok so to be clear, this PR introduces another breaking change: no more type checking?

Reply:
Yes, it removes out of the box support from running ncc. If we are considering that is a mandatory feature we need to support, I'm afraid it's not achievable via swc.

Reply:
Ok I update the PR title and description to mention this is semver major and has breaking changes.

Also, it seems there are still failing tests

Reply:
Huh, weird that only subset of node.js 16 CI's failing. I'll try to peek.